### ISSUE TITLE
'Subversion' object has no attribute 'info'

### ISSUE THREAD

**Issue by EddieDover** [wakatime/wakatime-cli#388]
Noticed an error today and upon examining the log, this is the latest line:
> {"now": "2021/05/19 11:43:59 -0700", "version": "13.0.7", "plugin": "vscode/1.56.2 vscode-wakatime/10.0.0", "time": 1621449839.8536325, "level": "ERROR", "message": "Traceback (most recent call last):\n  File \"wakatime\\main.py\", line 64, in execute\n  File \"wakatime\\heartbeat.py\", line 110, in __init__\n  File \"wakatime\\project.py\", line 94, in get_project_info\n  File \"wakatime\\projects\\subversion.py\", line 41, in folder\nAttributeError: 'Subversion' object has no attribute 'info'\n"}

Rolling back to 9.0.5 is a temporary fix? or at least seems to stop the error and lets wakatime continue uninterrupted.

**Comment by EddieDover** [comment]
> 
> 
> It might be fixed by the new [wakatime-cli](https://github.com/wakatime/wakatime-cli) based in Go. To try it out could you please add `alpha = true` to your `~/.wakatime.cfg` file? After restart vscode and it should work.

Works perfectly after following this step.

**Comment by alanhamlett** [comment]
You would also need this line in your `~/.wakatime.cfg` because we rolled back the Go wakatime-cli launch to fix some bugs:

    legacy_python_cli = true

**Comment by EddieDover** [comment]
This error has started happening again, 
```
[WakaTime][ERROR] Traceback (most recent call last):
  File "wakatime\main.py", line 64, in execute
  File "wakatime\heartbeat.py", line 110, in __init__
  File "wakatime\project.py", line 94, in get_project_info
  File "wakatime\projects\subversion.py", line 41, in folder
AttributeError: 'Subversion' object has no attribute 'info'
```
The fixes above don't change the result.
```
vscode/1.59.0
vscode-wakatime/14.0.3
```

**Comment by alanhamlett** [comment]
Does it only happen when you're coding offline? We still fallback to the legacy python cli when unable to send heartbeats with the new cli.

**Comment by EddieDover** [comment]
> 
> 
> Does it only happen when you're coding offline? We still fallback to the legacy python cli when unable to send heartbeats with the new cli.

This happens regardless of online status. I even switched away from my pihole dns server just to be sure I wasn't blocking it somehow. Weirdly enough it seems to be working fine when I'm using VSCode and ssh'd into a Linux server on the same network, but the native windows version hasn't connected in quite a while.  I've also tried uninstalling/reinstalling the extension and completely removing the wakatime folders to make sure I didn't have any old/cached installs.

**Comment by alanhamlett** [comment]
Oh, yes we rolled back the new Cli for a bug. We'll release a new version of vscode-wakatime that enables the new Cli again to fix this.

**Comment by alanhamlett** [comment]
Fixed in v15.0.0 of vscode wakatime extension.

**Comment by EddieDover** [comment]
> Fixed in v15.0.0 of vscode wakatime extension.

Thank you for your quick action. Everything is working perfectly after this update. 👍🏻
